APIKit/APIKit.swift

```diff
@@ -26,16 +26,12 @@ public enum Method: String {
     case CONNECT = "CONNECT"
 }
 
-public class API {
+public class API: NSObject, NSURLSessionDelegate {
     // configurations
     public class func baseURL() -> NSURL {
         return NSURL()
```

Comment:
This class func is practically abstract (returning empty URL is meaningless). So if the user doesn't override it, it should immediately crash with a message IMO.

And bacase `fatalError` is `@noreturn` function, we don't need to return some value.